src/sentry/tasks/commits.py

```diff
@@ -159,8 +160,10 @@ def fetch_commits(release_id: int, user_id: int, refs, prev_release_id=None, **k
                     "start_sha": start_sha,
                 },
             )
-            span = sentry_sdk.Hub.current.scope.span
-            span.set_status("unknown_error")  # type: ignore
+            span = sentry_sdk.get_current_span()
+            if not (isinstance(span, Span)):
+                raise TypeError("No span is currently active right now")
```

Comment:
`if span is None:` is the better way to write this -- or perhaps we are guaranteed to be in a span and you could assert it? (I think it would come from the decorator)

Reply:
I'll change it check for None; more often than not it should return a span but I think it's nice to have the check here so that if it does start to return None for the span, we'll be alerted